src/NuGet.Clients/NuGet.PackageManagement.UI/UserInterfaceService/INuGetUI.cs

```diff
@@ -95,7 +95,7 @@ public interface INuGetUI : IDisposable
         /// </summary>
         FileConflictAction FileConflictAction { get; }
 
-        IEnumerable<SourceRepository> ActiveSources { get; }
+        PackageSourceMoniker ActiveSources { get; }
```

Comment:
packageSourceMoniker has a package source name and IEnum<sources>, i think.
feels odd to have a singular type, but plural name here.
is there a naming problem with the type or the property?